qa/tasks/vstart_runner.py

```diff
@@ -271,8 +271,10 @@ def put_file(self, src, dst, sudo=False):
         try:
             shutil.copy(src, dst)
         except exception as e:
-            if sys.version_info.major < 3 and e.message.find('are the same '
-               'file') != -1:
+            if sys.version_info.major < 3:
+                if e.message.find('are the same file') != -1:
```

Comment:
This change is not required, previous code works fine.
In previous code, exception message is checked only if `sys.version_info.major < 3` is `True`

Reply:
Without this change code would be -

```
            if sys.version_info.major < 3 and e.message.find('are the same '
               'file') != -1:
           else:
                 return
```

If  `sys.version_info.major < 3` is True and ` e.message.find('are the same file') != -1` is False, the code under `else` would run which would be undesirable.

Reply:
See the suggestion below